src/typescript/13-utility-types/utility-types.md

```diff
@@ -123,180 +213,178 @@ let dinosCollection: Record<string, Dinosaur> = {
   },
 };
 ```
+@highlight 13
+
+In the code above, `dinosCollection` is equivalent to `{[key: string]: Dinosaur }`
 
-## Pick&lt;Type, Keys&gt;
+## Objective 6: Pick&lt;Type, Keys&gt;
 
-Selects only the properties defined in Keys. Useful if you want a subset of `Type`
+Select only the properties defined in Keys. Useful if you want a subset of `Type`.
 
 ```typescript
+enum Diet {
+  "Carnivore",
+  "Herbivore",
+  "Omnivore",
+};
+
+interface Dinosaur {
+  species: string;
+  diet: Diet;
+  age?: number; // the question mark signals that this property is optional
+};
+
 type LesserDinosaur = Pick<Dinosaur, "species" | "age">;
 
-let lesserDino: LesserDinosaur = {
+const lesserDino: LesserDinosaur = {
   species: "Tyrannosaurus rex",
   age: 27,
-  // diet: Diet.Carnivore, <- if this line was present, then the error below would be thrown by typescript
-  // Type '{ species: string; age: number; diet: Diet; }' is not assignable to type 'LesserDinosaur'.
-  //  Object literal may only specify known properties, and 'diet' does not exist in type 'LesserDinosaur'.
 };
 ```
+@highlight 13, 16, 17, only
+
+In the code above, if there is an attempt to add `diet` to `lesserDino` then TypeScript will throw an error. Object literal may only specify known properties, and `diet` does not exist in type `LesserDinosaur`.
 
-## Omit&lt;Type, Keys&gt;
+## Objective 7: Omit&lt;Type, Keys&gt;
```

Comment:
We’re going to have a string of three objectives:

- Extract: https://github.com/bitovi/trainings/blob/main/typescript-utility-types/Exercise2/solution/Solution.ts
- Exclude: https://github.com/bitovi/trainings/blob/main/typescript-utility-types/Exercise4/solution/Solution.ts
- Omit: https://github.com/bitovi/trainings/blob/main/typescript-utility-types/Exercise5/solution/Solution.ts

Extract and Exclude can be their own objectives because we have exercises for them.

For omit, I would call the objective something like “Include and exclude specific properties” and then have a concept each for Omit and Pick (since they go together). The exercise will only cover Omit, that’s fine.